lockbox-ios/Store/UserInfoStore.swift

```diff
@@ -5,6 +5,7 @@
 import Foundation
 import RxSwift
 import RxCocoa
+import UIKit
```

Comment:
None of the other `Store`s import UIKit. Think carefully about the need to import this framework here; if I were doing this implementation, I would 1) write a helper extension to `UIApplication` and 2) put _all_ calls to outside services in an `ActionHandler`. Stores work internally to the application; `ActionHandler`s do the required outside calls. 

tbh, looking at this class again I'm not even sure that the `AutoLockSetting` enum belongs here anymore either...